### PR TITLE
mongo: Increase the timeout for a connection

### PR DESCRIPTION
**Pull request by Igoraamc**
What does this implement/fix? Explain your changes.
---------------------------------------------------
This pull request implements unit tests for all interactor features.

- Adds the connection timeout configuration

Does this close any currently open issues?
------------------------------------------
Any relevant logs, error output, etc?
-------------------------------------
Any other comments?
-------------------
Where has this been tested?
---------------------------

**Operating System/Platform:** MACOS Catalina

**GO Version:** go 1.13.7 darwin/amd64

### CODE REVIEW

File: pkg/data/mongo.go [hunk 1]
```diff
@@ -2,12 +2,16 @@ package data
...
 	"context"
+	"time"
 
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 	"go.mongodb.org/mongo-driver/mongo/readpref"
 )
 
+// CONNECTIONTIMEOUT is a constant to set the waiting time for a database connection
+const CONNECTIONTIMEOUT = 60 * time.Second
```
Comment (netoax) [inline pkg/data/mongo.go:13]:
```suggestion
const CONNECTION_TIMEOUT = 60 * time.Second
```

**Review by netoax — CHANGES_REQUESTED** [review]
LGTM. I did just a simple suggestion.

File: pkg/data/mongo.go (hunk 1 above)
Comment (netoax) [inline pkg/data/mongo.go]: Earlier I suggested you naming it in upper case, sorry. In golang, it'd be expected to use `ConnectionTimeout` or `connectionTimeout`.

```suggestion
const connectionTimeout = 60 * time.Second
```

**Review by netoax — CHANGES_REQUESTED** [review]

File: pkg/data/mongo.go (hunk 1 above)
Comment (Igoraamc) [reply pkg/data/mongo.go]: Done!

**Pull request merged**